### ISSUE TITLE
Add actual onboarding for iOS

### ISSUE THREAD

**Issue by Odrakir** [toggl/mobileapp#7011]
This is about adding the actual onboarding steps to the onboarding screens.

Bear in mind that this will be the first screen the user will see when first opening the app, so take extra care here, everything has to be perfect, beautiful, animate when possible, load fast and generally be a pleasure to use.

We should also add analytics on what steps does the user view and more.

#6998 should be done first.

Designs and full specs: https://www.notion.so/toggl/Design-specs-for-onboarding-carousel-a55437219e074f6a869ab4f9ab8e5ff4

**NOTE:** This issue is part of the `Onboarding refactor` project and it should be merged into the project branch and not into develop.
